packages/typescript-to-proptypes/package.json

```diff
@@ -6,9 +6,6 @@
   "engines": {
     "node": ">=10.3.0"
   },
-  "files": [
-    "build"
-  ],
```

Comment:
Drive-by cleanup: Since this package isn't published, the `files` field is unused.